accessibility-swarms.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: page
+title: Accessibility Swarms
+---
+Accessibility Swarms are a practice that we carry out in BBC News.
+
+Swarms take place on **all new front-end code**, this means any changes to HTML, CSS or JavaScript code.
+
+We do this because **accessibility is a team responsibility**.
+
+## When should you carry out a swarm?
+- Before a ticket moves from Development to Test
+- When HTML, CSS & JS is code complete
+- Some developer testing with some assistive technology should already have taken place
+
+## How many people do you need?
+### If it’s a new component or feature
+Carry out a **team swarm**, with 4 to 6 people.
+
+### If it’s a small change
+Carry out a **developer swarm**.
+
+- You don’t need 4 to 6 people
+- The developers pairing on the ticket can work through the **[checklist](accessibility-news-and-testers)** alone
+- You only need to check what’s relevant. For example, if you have only changed a colour, then you only need to check what’s relevant to this. Such as, colour contrast and colour blindness.
```

Comment:
```suggestion
- You only need to check what’s relevant. For example, if you have only changed a colour, then you only need to check what’s relevant to this, e.g. colour contrast and colour blindness.
```

Reply:
Thanks @ChrisBAshton http://www.hemingwayapp.com/ suggested that a new sentence was better for readability.